.changesets/a6vql.minor.md

```diff
@@ -0,0 +1,2 @@
+Feat: Add `id_token` to the return value of LinkedIn's `validateAuthorizationCode(code: string)`   
```

Comment:
Can you make a changeset for each change (feat one should be minor and the fix should be patch)